pkg/selinux/selinux-client.js

```diff
@@ -96,6 +102,66 @@ export function init(statusChangedCallback) {
         );
     };
 
+    var parseBoolean = function(result, item) {
+        // Example:
+        // authlogin_nsswitch_use_ldap (on , on) Allow authlogin to nsswitch use ldap
+        // Split by first ')', as the name cannot contain ')'
+        result.push(item.split(")", 2)[1].trim());
+        return result;
+    };
+
+    var getModifications = function() {
+        // List of items we know how to parse
+        let manageditems_callbacks = [["boolean", parseBoolean]];
+        let manageditems = manageditems_callbacks.map(item => item[0]);
+
+        // Building a query to get information from semanage
+        // Use `semanage export` to show shell script (and parse types we yet don't parse explicitly)
+        // Use `semanage <type> --list -C` to get better readable and parsable local changes
+        // Use `echo '~~~~~'` as separator, so we don't need to execute multiple commands
+        let script = "semanage export";
+        manageditems.forEach(item => { script += " && echo '~~~~~' && semanage " + item + " --list -C --noheading" });
+        cockpit.spawn(["/bin/sh", "-c", script], { err: 'message', environ: [ "LC_ALL=C" ], superuser: "try" }).then(
```

Comment:
You can use `cockpit.script` for this. Also, please set `LC_MESSAGES` only, so that this doesn't damage non-ASCII file names.

Put the `.then()` on a new line and use an arrow instead of `function(output)` so that this becomes bit tidier.